src/applications/check-in/pre-check-in/pages/Validate/index.jsx

```diff
@@ -58,10 +69,24 @@ export default function Index({ router }) {
         goToNextPage();
       } catch (e) {
         setIsLoading(false);
-        goToErrorPage();
+        if (
```

Comment:
Not in this PR, but I think we can do some refactoring around the API calls and duplication of logic. Just thinking out loud